assets/js/editor.js

```diff
@@ -24,6 +24,23 @@ $gp.editor = (
 			translation_id_from_row_id: function( row_id ) {
 				return row_id.split( '-' )[ 1 ];
 			},
+			update_count: function( element ) {
+				var id = $( element.target ).attr( 'id' ).substring( 'translation_'.length );
+				var string = $( 'textarea#translation_' + id ).val();
+				var characters_count = string.length;
+				var words_count = 0;
+
+				if ( characters_count > 0 ) {
+					if ( string.indexOf( ' ' ) !== -1 ) {
+						words_count = string.trim().split( ' ' ).length;
+					} else {
+						words_count = 1;
+					}
+				}
```

Comment:
Should we make use of https://github.com/WordPress/gutenberg/tree/trunk/packages/wordcount?

Reply:
It could be helpful, but it also means bumping JS to ES2015, adding the first JS dependency, and updating the deployment to include it.

Reply:
Done, the PR is using Wordcount package.